### ISSUE TITLE
about the training and testing dataset

### ISSUE THREAD

**Issue by ShenghuaCheng** [prov-gigapath/prov-gigapath#37]
If you can clarify that the dataset used to pretrain Prov-GigaPath is strictly seperated from the internal test set and TCGA external test set. Thank you!

**Comment by HanwenXuTHU** [comment]
Hi, thank you for reaching out. We followed the standard setting suggested by HIPT and other prior work. Our pretraining data from Providence included all WSIs and tiles, without any labels or metadata. No external data, including TCGA images, were used for pretraining. For further details, please refer to our paper.

**Comment by HanwenXuTHU** [comment]
Close this issue for now. Please let us know if you have further question!

**Comment by ShenghuaCheng** [comment]
Thank you for your reply.  In fact, i want to know if it's possible that the Providence images and TCGA images overlap with each other, as both of them are very huge dataset.

**Comment by HanwenXuTHU** [comment]
Hi, the Providence images are from our internal dataset Prov-Path and there is no overlap with TCGA images.

**Comment by ShenghuaCheng** [comment]
I get it. really thank you !
